src/osd/ECBackend.cc

```diff
@@ -1891,7 +1895,7 @@ bool ECBackend::try_reads_to_commit()
   waiting_commit.push_back(*op);
 
   dout(10) << __func__ << ": starting commit on " << *op << dendl;
-  dout(20) << __func__ << ": " << cache << dendl;
+  //dout(20) << __func__ << ": " << cache << dendl;
```

Comment:
We don't really want to be commenting these out.  Maybe make it dout(30) or remove it?